pkg/cloud/instance.go

```diff
@@ -282,7 +282,7 @@ func (c *client) GetOrCreateVMInstance(
 		return err
 	}
 	csMachine.Spec.InstanceID = pointer.StringPtr(deployVMResp.Id)
-
+	csMachine.Status.Status = metav1.StatusSuccess
```

Comment:
will this cause a panic for older machines which have `csMachine.Status = nil`?

Reply:
csMachine.Status is a struct in previous version, so it cannot be nil.